### ISSUE TITLE
Missing APPS data

### ISSUE THREAD

**Issue by danijeljw** [oseyx/SOE_Staging#41]
```powershell
Copy-Item : Cannot find path 'G:\SOE_Staging\IVCApps\SOE_Config\APPS' because it does not exist.
```

**Comment by danijeljw** [comment]
I accidentally deleted the WHOLE application folder!!!

WTF!?!?!?!

**Comment by danijeljw** [comment]
Reconstructed using Latitude 5290. Now it's also more modular. Amazing.

**Comment by danijeljw** [comment]
Major Updates
Testing #41
Resolves #40
Testing #34
Testing #33
Resolves #27
Resolves #19
Resolves #3

**Comment by danijeljw** [comment]
The APPS data has already been resolved. It now reads directly from SCCM. Add this to the documentation only.

Marked in issue #51 for more details
